include/openthread/platform/time.h

```diff
@@ -52,17 +52,34 @@ extern "C" {
  */
 
 /**
- * Get the current time (64bits width).
+ * Get the current time in microseconds referenced to a continuous monotonic
+ * local radio clock (64 bits width).
+ *
+ * The radio clock SHALL not wrap during the device's uptime. Implementations
+ * SHALL therefore identify and compensate for internal counter overflows. The
+ * clock does not have a defined epoch nor SHALL it introduce any continuous or
```

Comment:
Correction on normative requirement language:

```suggestion
 * clock does not have a defined epoch and SHALL NOT introduce any continuous or
```

Reply:
Fixed.